### DISCUSSION TITLE
Beatmap carousel scrolls back when near the scoreboard or beatmap information

### DISCUSSION THREAD

**Discussion by AlphanTex** [ppy/osu#30452] [Q&A]
### Type

Cosmetic

### Bug description

The scoreboard and beatmap information seem to have incorrect hitboxes making song select scroll back to the selected song too early.
There is also no deadzone as with stable. (not sure if that's intentional).
I have included a screenshot and a video demonstrating the issue.

### Screenshots or videos

Video:
https://github.com/user-attachments/assets/bc909d17-a4a4-4053-a973-3590e7f4fd9a

And screenshot:
![Screenshot 2024-10-29 123011](https://github.com/user-attachments/assets/47169a7b-99c9-4761-af86-ef7c515eed36)

### Version

2024.1009.1

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17556382/compressed-logs.zip)

**Comment by SchiavoAnto** [comment]
I didn't really understand the issue, could it be related to #24055?

**Comment by peppy** [comment]
The line you show looks about correct. Are you hoping for it to only happen when you actually hover the leaderboard or something?

**Reply by AlphanTex** [reply]
Yes, as it's pretty disorienting for me to have it scroll back without my cursor hovering over the leaderboard, and I've talked to a few people who find it frustrating (altough most people find it a minor annoyance or a non-issue).
